lib/aullar/flair.moon

```diff
@@ -35,8 +35,21 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    if flair.custom_draw and
+         pcall flair.custom_draw, flair, x, y, width, height, cr
+      return
+    else
+      -- Draw an intensely red "shit has gone bad"-cross instead
```

Comment:
Ah, sorry, I've been reminded on many occasions my "private language" is less than appealing to many. I'll change it to more factual wording.